src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -185,12 +185,19 @@ export const createNewConcept = (data, dataUrl) => async (dispatch) => {
     dispatch(addConceptToDictionary(response.data.id, dataUrl));
     notify.show('creating concept, please wait...', 'warning', 3000);
   } catch (error) {
-    notify.show('concept could not be created, uuid must be unique', 'error', 3000);
     if (error.response) {
-      dispatch(isErrored(error.response.data, CREATE_NEW_CONCEPT));
+      const { response } = error;
+      notify.show(` There was error in creating a concept.
```

Comment:
The above grammar is not correct.

Reply:
Thanks for the feedback. I will update it.

Reply:
I have updated the description.